supabase/config.toml

```diff
@@ -0,0 +1,72 @@
+# A string used to distinguish different Supabase projects on the same host. Defaults to the working
```

Comment:
nit(non-blocking)
I'm guessing the config is used as part of supabase cli? Does it need to be a top-level folder here? Also does it need to be a toml file or can we use ts/js?

Reply:
That's right. I've found the supabase docs to be pretty good for what they cover, but they are often missing examples or explanations of alternative ways of doing things. But it does seem like the file has to be .toml and that it has to be in a `supabase` folder at the top-level. It is automatically generated by running [supabase init](https://supabase.com/docs/reference/cli/supabase-init), and other supabase functionality fails if the .toml file is not found at that location. Further docs on the config file [here](https://supabase.com/docs/reference/cli/config).

Perhaps it would be desirable to not commit this folder, and have authors and devs run `supabase init` themselves as part of the set-up? Although if we end up extending supabase functionality and changing parts of the config from the default, then we would want the file to be committed.

Reply:
Given that this is only for interacting with CLI I don't expect authors will need to do anything with this folder. Nothing in this folder will be available to the runtime app, and is just for running a local server. I expect even developers running the app locally may choose to simply connect to an online dev project instead of running their own postgres server locally.

So I'd say probably safe to ignore and add developer docs if we find we want to support people running cli locally in the future